### ISSUE TITLE
Experiment: check trainset performance without dropout

### ISSUE THREAD

**Issue by MaxPowerWasTaken** [MaxPowerWasTaken/voice_to_text#4]
following up on results from #1, and this advice from one of the Mozilla DeepSpeech maintainers:

> For example depending on how “noisy” your data is you should tune dropout_rate up or down. The more “noisy” the lower dropout_rate should be as the data regulates the network itself. I don’t know what the maximal value of dropout_rate should be for “clean” data and I don’t know what the minimum should be for “noisy” data.
https://discourse.mozilla.org/t/recommended-hyperparameters-range-for-fine-tuning/30048

...running:
```
    python DeepSpeech.py \
        --n_hidden 2048 \
        --dropout_rate 0.01 \
        --checkpoint_dir ../../model_retrained2/deepspeech-0.5.1-checkpoint/ \
        --epochs 50 \
        --early_stop False \
        --train_files ../../data/ds_csvs/train.csv \
        --test_files ../../data/ds_csvs/train.csv \
        --learning_rate 0.0001 \
        --train_batch_size 6 \
        --test_batch_size 4 \
        --lm_binary_path models-0.5.1/lm.binary \
        --lm_trie_path models-0.5.1/trie \
        --export_dir ../../model_retrained2
```
(as you can see at https://github.com/mozilla/DeepSpeech/blob/master/util/flags.py , there's actually 6 dropout_rate* params, but half default to 0.0 and half default to `dropout_rate`)

**Comment by MaxPowerWasTaken** [comment]
not really much better, even on trainset
```
Epoch 48 |   Training | Elapsed Time: 0:00:42 | Steps: 10 | Loss: 81.479848  
Epoch 49 |   Training | Elapsed Time: 0:00:42 | Steps: 10 | Loss: 79.408006  
I FINISHED optimization in 0:34:22.297876
I Restored variables from most recent checkpoint at ../../model_retrained2/deepspeech-0.5.1-checkpoint/train-467856, step 467856
Testing model on ../../data/ds_csvs/train.csv
Test epoch | Steps: 15 | Elapsed Time: 0:00:29                               
Test on ../../data/ds_csvs/train.csv - WER: 0.692437, CER: 0.594374, loss: 74.483017
--------------------------------------------------------------------------------
WER: 1.000000, CER: 0.781250, loss: 69.854370
 - src: "he emailed me yesterday about it"
 - res: "i have a time"
--------------------------------------------------------------------------------
WER: 1.000000, CER: 0.775000, loss: 71.680199
 - src: "we're stopping water yoga for the summer"
 - res: "we too or"
--------------------------------------------------------------------------------
WER: 1.000000, CER: 0.619048, loss: 74.006088
 - src: "neal emailed them but hasnt heard back yet"
 - res: "well he thither at"
--------------------------------------------------------------------------------
WER: 1.000000, CER: 0.804878, loss: 79.962387
 - src: "your hair and skin both look really great"
 - res: "yet in the"
--------------------------------------------------------------------------------
WER: 1.000000, CER: 0.862745, loss: 100.686104
 - src: "do you remember where you put yesterday's newspaper"
 - res: "no one had"
--------------------------------------------------------------------------------
WER: 1.000000, CER: 0.830508, loss: 110.470367
 - src: "neal left to pick up max at the silver spring metro station"
 - res: "let it he "
--------------------------------------------------------------------------------
WER: 1.000000, CER: 0.904762, loss: 117.595306
 - src: "are your two children still living in new york and philadelphia"
 - res: "a a a a"
--------------------------------------------------------------------------------
WER: 1.000000, CER: 0.916667, loss: 121.725960
 - src: "terri swam laps today at the community swimming pool despite the weather"
 - res: "tester"
--------------------------------------------------------------------------------
WER: 0.937500, CER: 0.658537, loss: 125.984016
 - src: "lilah watched the workers from a safe corner on the wet floor of the packing plant"
 - res: "li t o o t o o t o o t o o plant"
--------------------------------------------------------------------------------
WER: 0.923077, CER: 0.650794, loss: 97.777321
 - src: "let's get a bite to eat after the concert at the kennedy center"
 - res: "let it go at that then "
--------------------------------------------------------------------------------
I Exporting the model...
WARNING:tensorflow:From /home/mepstein/.local/share/virtualenvs/voice_to_text-2wIsmzYN/lib/python3.6/site-packages/tensorflow/python/tools/freeze_graph.py:232: convert_variables_to_constants (from tensorflow.python.framework.graph_util_impl) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.compat.v1.graph_util.convert_variables_to_constants
WARNING:tensorflow:From /home/mepstein/.local/share/virtualenvs/voice_to_text-2wIsmzYN/lib/python3.6/site-packages/tensorflow/python/framework/graph_util_impl.py:245: extract_sub_graph (from tensorflow.python.framework.graph_util_impl) is deprecated and will be removed in a future version.
Instructions for updating:
Use tf.compat.v1.graph_util.extract_sub_graph
I Models exported at ../../model_retrained2```
